utensor_cgen/backend/utensor/code_generator/rearch/_operators/_impls.py

```diff
@@ -799,5 +836,5 @@ def get_eval_snippet(self, op_var_name, op_info, tensor_var_map):
   def get_construct_snippet(self, op_var_name):
     return None
 
-
-OperatorFactory._operators[_MissingOperator.op_type] = _MissingOperator
+for namespace in _MissingOperator.namespaces:
```

Comment:
Why do you need to add these loop?
It will break the code generator.

Reply:
Oh, I think of a better way to do this.
Please revert this loop.
I'll make update to `develop` branch.

Reply:
Added #136 to track the fix

Reply:
Reverted